great_expectations/checkpoint/actions.py

```diff
@@ -217,6 +196,10 @@ def __init__(  # noqa: PLR0913
         self.notify_with = notify_with
         self.show_failed_expectations = show_failed_expectations
 
+        from great_expectations import project_manager
```

Comment:
Can we move this import to the top of the file? If not, could you add a comment.

Similar comment at other places in this file.